docs/layout-doc.md

```diff
@@ -77,8 +80,9 @@ Mostly used for debugging and testing, it can take a color so the debugger will
 
 
 ### ESizingLayout
-This layout is used to override the size of its child in different ways: overwriting width, overwriting height, scaling, and a few more.
+This layout is used to override the size of its child in different ways: width, height, scaling, and a few more.
 
+//TODO - It's sorta hard but it might worth coming up with a new word instead of snug. Doesn't work here
 ### ESnuggingLayout
 
 Snugs a layout on its axis, the equivalent of Android's wrap_content feature. However it can only be used on layouts that implement `ELayoutAlongAxis`:
```

Comment:
It's sorta hard but it might worth coming up with a new word instead of "snugs". Doesn't work here